tensorflow/lite/kernels/tile_test.cc

```diff
@@ -56,192 +56,72 @@ class TileOpModel : public SingleOpModel {
   int output_;
 };
 
-TEST(TileTest, Float32Vector) {
-  TileOpModel m({3}, TensorType_FLOAT32, TensorType_INT32);
-  m.SetInput<float>({1.f, 2.f, 3.f});
-  m.SetMultipliers({2});
+template <typename T>
+void Check(std::initializer_list<int> input_shape,
+           std::initializer_list<T> input_data,
+           std::initializer_list<int> multipliers_data,
+           std::initializer_list<int> exp_output_shape,
+           std::initializer_list<T> exp_output_data, TensorType input_type,
+           TensorType multiply_type) {
+  TileOpModel m(input_shape, input_type, multiply_type);
+  m.SetInput(input_data);
+  m.SetMultipliers(multipliers_data);
   m.Invoke();
-  EXPECT_THAT(m.GetOutput<float>(),
-              ElementsAreArray({1.f, 2.f, 3.f, 1.f, 2.f, 3.f}));
+
+  EXPECT_THAT(m.GetOutputShape(), ElementsAreArray(exp_output_shape));
+  EXPECT_THAT(m.GetOutput<T>(), ElementsAreArray(exp_output_data));
+}
+
+TEST(TileTest, Float32Vector) {
+  Check<float>({3}, {1.0, 2.0, 3.0}, {2}, {6}, {1.0, 2.0, 3.0, 1.0, 2.0, 3.0},
```

Comment:
Add comments to the inputs and make it more clear, such as /* input_shape =*/ {3}.

Reply:
comments added to inputs.